.github/workflows/build.yml

```diff
@@ -64,28 +64,14 @@ jobs:
         if: matrix.os == 'macos-13' || matrix.os == 'macos-14'
         run: |
           brew install make clang-format
-      # The build step includes :spotlessJavaBuild.
-      - name: Build Cooja and Documentation
-        uses: gradle/gradle-build-action@v3
-        with:
-          gradle-version: 8.7
-          cache-read-only: false
-          arguments: build -Perrorprone
-          build-root-directory: contiki-ng/tools/cooja
-      # The Jar is important for MSPSim in Contiki-NG.
-      - name: Build Jar
-        uses: gradle/gradle-build-action@v3
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
         with:
-          gradle-version: 8.7
           cache-read-only: false
-          arguments: jar
-          build-root-directory: contiki-ng/tools/cooja
-      - name: Test MSPSim
-        uses: gradle/gradle-build-action@v3
-        with:
-          gradle-version: 8.7
-          arguments: test
-          build-root-directory: contiki-ng/tools/cooja
+      # The build step includes :spotlessJavaBuild.
+      - name: Build and test
+        working-directory: contiki-ng/tools/cooja
+        run: ./gradlew build jar -Perrorprone
```

Comment:
This does not run `./gradlew test`. (We only have one test right now and that test is pretty useless, but I think we want to keep it wired in so it's easy to add more tests.)

Reply:
As far as I understand, `./gradlew build` do run the test. This is the log from the CI for this PR.

```
> Task :compileJava

> Task :processResources
> Task :classes
> Task :fullJar
> Task :copyDependencies
> Task :jar
> Task :startScripts
> Task :distTar
> Task :distZip
> Task :assemble
> Task :spotlessInternalRegisterDependencies
> Task :spotlessJava
> Task :spotlessJavaCheck
> Task :spotlessCheck
> Task :compileTestJava
> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test

TestCooja > testVersion() PASSED

> Task :check
> Task :build
```

Reply:
The log is surprising to me, but there's no doubt the tests run as you say.